scripts/game/GameScores.js

```diff
@@ -20,19 +22,47 @@ const render = (team1ID, team2ID, team3ID) => {
     <div class="score--input--form">
         <fieldset class="input--container">
             <label for="score1">${team1Obj.name}</label>
-            <input name="score1" class="score--input" type="text" placeholder="Enter score..."></input>
+            <!-- string interpolate something? maybe? team id? beuller?-->
```

Comment:
I'm not sure what this comment means. We have the team name on the line above. If this comment isn't useful, it should be removed.